lib/kubernetes-deploy/runner.rb

```diff
@@ -105,9 +105,9 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
 
       if deploy_has_priority_resources?(resources)
         @logger.phase_heading("Predeploying priority resources")
-        ::StatsD.measure('duration.priority_resources', tags: statsd_tags) do
-          predeploy_priority_resources(resources)
-        end
+        start_priority_resource = Time.now.utc
+        predeploy_priority_resources(resources)
+        ::StatsD.measure('duration.priority_resources', (Time.now.utc - start_priority_resource), tags: statsd_tags)
```

Comment:
Should we round this number (and the other two)? If we don't we have the slight weirdness that the individual resource times can appear to be longer than this time because of the extra resolution here. 

```
I, [2017-07-12T14:09:52.499086 #10373]  INFO -- : [StatsD] measure KubernetesDeploy.resource.duration:0.3 #context:minikube #namespace:k8sdeploy-test-deploy-result-logging-for-mixed-dcae97ad348d6836 #resource:ConfigMap/hello-cloud-configmap-data #type:ConfigMap #sha: #status:success
I, [2017-07-12T14:09:52.499495 #10373]  INFO -- : [StatsD] measure KubernetesDeploy.duration.priority_resources:0.270041 #namespace:k8sdeploy-test-deploy-result-logging-for-mixed-dcae97ad348d6836 #sha:5698f34b7826 #context:minikube
```